cattleshed-conf/compilers.py

```diff
@@ -1922,14 +1922,15 @@ def make_go(self):
                 'displayable': True,
                 'language': 'Go',
                 'output-file': 'prog.go',
-                'compiler-option-raw': False,
+                'compiler-option-raw': True,
                 'compile-command': ['/bin/true'],
                 'version-command': version_command,
                 'switches': [],
                 'initial-checked': [],
                 'display-name': display_name,
-                'display-compile-command': 'go run prog.go',
-                'run-command': ['/opt/wandbox/go-{cv}/bin/go', 'run', 'prog.go'],
+                'display-compile-command': ['go', 'build', '-o', 'prog', 'prog.go'],
+                'compile-command': ['/opt/wandbox/go-{cv}/bin/go', 'build', '-o', 'prog', 'prog.go'],
```

Comment:
Compiler options can only be appended to the end of `compile-command`. So remove `prog.go`.